src/styles/custom.less

```diff
@@ -47,3 +47,7 @@
 @layout-trigger-height       : 48px;
 @layout-zero-trigger-width   : 36px;
 @layout-zero-trigger-height  : 42px;
+
+// Screen Sizes
+@medium-screen: 990px;
```

Comment:
We already have existing value for screensize see here: https://github.com/busyorg/busy/blob/702bc7cb698fb8f200a768024cfbc0285258ad13/src/styles/modules/variables.less#L7-L9
I think we the responsive layout should be triggered when size is lower than `768px`. Similar than right sidebar or search in topnav.

Reply:
we can't use these properties, these are set to be `min-width:` rather than `max-width`